service/worker/processor.go

```diff
@@ -199,7 +199,10 @@ func (p *replicationTaskProcessor) processWithRetry(msg kafka.Message, workerID
 	forceBuffer := false
 	remainingRetryCount := p.config.ReplicationTaskMaxRetry
 
+	attempt := 0
```

Comment:
We already have an attempt variable as part of for loop.  Can we just have a one instead of 2?

Reply:
Probably you also want to include attempt on warn logs for every 100 failures.

Reply:
all done